src/styles/typography.scss

```diff
@@ -0,0 +1,97 @@
+@import './mixins';
+
+@function rem-calc($size) {
+  $remSize: $size / 16;
+  @return #{$remSize}rem;
+}
+
+@mixin font($style) {
+  @if ($style == "DS-60") {
+    font-size: rem-calc(12);
+    line-height: 1.5;
+  }
+
+  @if ($style == "DS-80") {
+    font-size: rem-calc(12);
+    line-height: 1.334;
+  }
+
+  @if ($style == "DS-100") {
+    font-size: rem-calc(16);
+    line-height: 1.5;
+  }
+
+  @if ($style == "DS-200") {
+    font-size: rem-calc(20);
+    line-height: 1.6;
+  }
+
+  @if ($style == "DS-300") {
+    font-size: rem-calc(24);
+    line-height: 1.334;
+  }
+
+  @if ($style == "DS-400") {
+    font-size: rem-calc(32);
+    line-height: 1.25;
+  }
+
+  @if ($style == "DS-500") {
+    font-size: rem-calc(40);
+    line-height: 1.2;
+  }
+
+  @if ($style == "DS-600") {
+    font-size: rem-calc(48);
+    line-height: 1.1667;
+  }
+
+  @if ($style == "DS-700") {
+    font-size: rem-calc(64);
+    line-height: 1.125;
+  }
+
+  @if ($style == "DS-800") {
+    font-size: rem-calc(76);
+    line-height: 1.157;
+  }
+}
+
+@mixin document-typography($scale) {
+  font-size: 100%;
+  line-height: 1.65;
+
+  p {
+    margin-bottom: 1.15rem;
+  }
+
+  h1, h2, h3, h4, h5 {
+    margin: 2.75rem 0 1.05rem;
+    line-height: 1.15;
+  }
+
+  h1 {
+    margin-top: 0;
+    font-size: pow($scale, 5) * 1em;
+  }
+
+  h2 {
+    font-size: pow($scale, 4) * 1em;
+  }
+
+  h3 {
+    font-size: pow($scale, 3) * 1em;
+  }
+
+  h4 {
+    font-size: pow($scale, 2) * 1em;
+  }
+
+  h5 {
+    font-size: $scale * 1em;
+  }
+
+  small, .text_small {
+    font-size: pow($scale, -1) * 1em;
+  }
+}
```

Comment:
This whole logic is a little bit disconnected from our design system because it doesn't make use of the `font("DS-N")` mixin, it's there just to allow using other scales.
I suggest to remove the `document-typography` mixing but keep the style definitions of the headings and the paragraph element, that can be useful to have some styles our of the box, bu they should use the `font("DS-N")` mixin, e.g.
```scss
h1 {
  @include font("DS-800");
}
```

And I don't think we need to touch the margins.

Reply:
Don't you think it's a nice to have for a fast prototyping?